src/driver/DriverUtils.ts

```diff
@@ -127,14 +127,16 @@ export class DriverUtils {
         buildOptions: { shorten?: boolean; joiner?: string } | undefined,
         ...alias: string[]
     ): string {
-        const newAlias =
-            alias.length === 1
-                ? alias[0]
-                : alias.join(
-                      buildOptions && buildOptions.joiner
-                          ? buildOptions.joiner
-                          : "_",
-                  )
+        const joiner =
+            buildOptions && buildOptions.joiner ? buildOptions.joiner : "_"
+
+        let newAlias = alias.length === 1 ? alias[0] : alias.join(joiner)
+
+        const hasNestedRelations = newAlias.includes(".")
+        if (hasNestedRelations) {
+            newAlias = newAlias.replace(/\./g, joiner)
```

Comment:
Are there some edge cases like dots in table names that would be affected by this? At least in postgres, I am able to create tables with dots and then use double quote marks to show that this is a name.

Reply:
That does sound like some edge case that could bring up some issue 🤔 . Haven't really worked with it. How does postgress handles the syntax for nested relations?

Reply:
Just removed this section that previously added, it was not needed at all